finagle-core/src/main/scala/com/twitter/finagle/client/MethodBuilder.scala

```diff
@@ -237,6 +240,18 @@ final class MethodBuilder[Req, Rep] private[finagle] (
    *
    * @note See `idempotent` below for a version that takes a [[Tunable[Double]]] for `maxExtraLoad`.
    */
+  def idempotent(
+    maxExtraLoad: Double,
+    sendInterrupts: Boolean,
+    minSendBackupAfterMs: Int,
+    classifier: service.ResponseClassifier
+  ): MethodBuilder[Req, Rep] = {
+    val brfParam =
+      if (maxExtraLoad == 0) BackupRequestFilter.Disabled
+      else BackupRequestFilter.Configured(maxExtraLoad, sendInterrupts, minSendBackupAfterMs)
+    addBackupRequestFilterParamAndClassifier(brfParam, classifier)
+  }
+
   def idempotent(
```

Comment:
since we are already here, would you mind adding the comments to the existing `idempotent` method, too? Thank you!
Also we could have one method call another here, too, to avoid duplicate code.

Reply:
The only problem I had with reusing these methods is that I would need to specify default value for **minSendBackupAfterMs** in idempotent method. For example :
```
def idempotent(
    maxExtraLoad: Double,
    sendInterrupts: Boolean,
    minSendBackupAfterMs: Int,
    classifier: service.ResponseClassifier
  ): MethodBuilder[Req, Rep] = {
    val brfParam =
      if (maxExtraLoad == 0) BackupRequestFilter.Disabled
      else BackupRequestFilter.Configured(maxExtraLoad, sendInterrupts, minSendBackupAfterMs)
    addBackupRequestFilterParamAndClassifier(brfParam, classifier)
  }

  def idempotent(
    maxExtraLoad: Double,
    sendInterrupts: Boolean,
    classifier: service.ResponseClassifier
  ): MethodBuilder[Req, Rep] =
    idempotent(maxExtraLoad, sendInterrupts, 1, classifier)
```
We would end up having "default" value for this parameter in two places. When calling idempotent method or when creating Configured object. We can always define something like and reuse it.
`private[finagle] val DefaultMinSendBackupAfterMs: Int = 1`
WDYT?